.env.example

```diff
@@ -0,0 +1,3 @@
+# override the default integration test url.
```

Comment:
Let's either place this file in /test if possible (I think not, not sure) or make it very clear these env vars are for testing.

Reply:
I feel like the way we are handling env vars goes against 12 factor app best practices by using a config file in the user's home directory. It makes sense when just looking at a developer's environment, but I don't feel like this is going to be scalable across additional environments (staging, prod, etc) In a container based environment, the env variables need to be passed in via docker.

I see an approach that uses .env to hold the overrides that are currently come from ~/.po.et/configuration. And then .env.test would contain those plus the additional env vars needed for testing.

cc: @WesleyCharlesBlake